source/audio/soundSplit.py

```diff
@@ -139,22 +141,41 @@ class VolumeSetter(AudioSessionNotification):
 
 	def on_session_created(self, new_session: AudioSession):
 		pid = new_session.ProcessId
+		process = new_session.Process
+		if process is not None:
+			exe = os.path.basename(process.exe())
+			isNvda = exe.lower() == "nvda.exe"
+		else:
+			isNvda = False
 		channelVolume = new_session.channelAudioVolume()
 		channelCount = channelVolume.GetChannelCount()
 		if channelCount != 2:
 			log.warning(f"Audio session for pid {pid} has {channelCount} channels instead of 2 - cannot set volume!")
 			self.foundSessionWithNot2Channels = True
 			return
-		if pid != globalVars.appPid:
-			channelVolume.SetChannelVolume(0, self.leftVolume, None)
-			channelVolume.SetChannelVolume(1, self.rightVolume, None)
-		else:
+		if pid == globalVars.appPid:
 			channelVolume.SetChannelVolume(0, self.leftNVDAVolume, None)
 			channelVolume.SetChannelVolume(1, self.rightNVDAVolume, None)
+		elif isNvda:
+			# This might be NVDA running on secure screen; don't adjust its volume
+			pass

```

Comment:
this should probably use `utils.security.isRunningOnSecureDesktop` if this is the case

Reply:
No. In my understanding `utils.security.isRunningOnSecureDesktop` tells whetehr current process is running on secure desktop. Here we assume that we're running in a normal mode, and here we're checking volume of other processes, and once we encounter another process called `nvda.exe` we assume it must be running on secure screen and we don't want to apply sound split to it.

Reply:
I fully agree with @mltony's answer here. We are not testing the current NVDA process but another NVDA process which is not the current one.

By the way, it makes me think to other questions and tests to do:
* What happens if I adjust the apps volume while VLC is playing audio in the session of another user in the background? (I have not tested if it is only possible...)
* What happens while another instance of NVDA is running in the session of another user opened in the background if I modify the apps volume through my current session's NVDA?
* Does other apps volume (or mute) work on secure screens? It would be useful in some cases when trying to log in while audio is being played in a session opened in the background.